### DISCUSSION TITLE
Template for name/display secondary attribute

### DISCUSSION THREAD

**Discussion by uXo7** [RomRider/apexcharts-card#117] [Q&A]
Hi guys,
is it possible to extend name with additional attribute? or using templates? Use case: Display graph of CO2 in every room and add to current state in header information about last datetime when the data were captured. Example: Netatmo is updating values in 10min interval.

**Comment by RomRider** [comment]
That's not a feature, but I can probably add it natively to the card.

**Reply by uXo7** [reply]
Thanks, that would be great!

**Reply by marlarius** [reply]
I need this feature too. It would be awesome if it was possible to show a sensor value in the title. Maybe a template, like {{ sensor.mysensor.value }} etc.
